2022-01-01 12:51:06,773 INFO: Initialized curve data.
2022-01-01 12:51:06,776 INFO: Setting up initial twist data...
2022-01-01 12:51:07,721 INFO: Twist setup complete.
2022-01-01 12:51:07,721 INFO: Searching for rational points on each twist...
2022-01-01 12:51:08,655 INFO: Finished searching for rational points on each twist.
2022-01-01 12:51:08,655 INFO: Testing local solvability of twists...
2022-01-01 12:51:13,882 INFO: Finished local solvability testing.
2022-01-01 12:51:13,885 INFO: Computing elliptic curve a-invariants...
2022-01-01 12:51:19,596 INFO: Finished computation of a-invariants.
2022-01-01 12:51:19,598 INFO: Computing MW group (delta = ['-1/2', '1/2', '1'], g = [4, -3, -12, 0, 12, 8])...
2022-01-01 13:33:09,548 ERROR: An exception occurred. Recording data and exiting.
Traceback (most recent call last):
  File "/project/arithgeo/drhast/twocover-descent/./twocover-processor.sage.py", line 506, in <module>
    rank, MW_proven, orders, gens = twist_MW_group(D)
  File "/project/arithgeo/drhast/twocover-descent/./twocover-processor.sage.py", line 327, in twist_MW_group
    A, mw, bool_rank, bool_index = magma.MordellWeilGroup(E, nvals=_sage_const_4 )
  File "/share/pkg.7/sagemath/9.3/install/lib/python3.9/site-packages/sage/interfaces/magma.py", line 1787, in __call__
    return M.function_call(self._name,
  File "/share/pkg.7/sagemath/9.3/install/lib/python3.9/site-packages/sage/interfaces/magma.py", line 1162, in function_call
    return self._do_call(fun, nvals)
  File "/share/pkg.7/sagemath/9.3/install/lib/python3.9/site-packages/sage/interfaces/magma.py", line 1221, in _do_call
    raise TypeError("Error executing Magma code:\n%s" % out)
TypeError: Error executing Magma code:

Current total memory usage: 7393.5MB, failed memory request: 1151.3MB
System Error: User memory limit has been reached
2022-01-03 13:14:58,000 INFO: Obsolete "loc_solv_error" keys removed.
